style.css

```diff
@@ -0,0 +1,131 @@
+/*  here is the main parent of html-style  */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/*  mobile menu bar  */
+#toolbar {
+  position: sticky;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 130px;
+  background-color: #3c3a39;
+  width: 377px;
+  height: 48px;
+  color: #fff;
+}
+
+/*  user name at the menu  */
+.name_div {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+/* the mobile menu icon */
+.menu_icon {
+  height: 24px;
+  width: 24px;
+}
+
+/*  headlines contaner  */
+#headlines {
+  display: grid;
+  height: 623px;
+  width: 377px;
+  border-radius: 0;
+  background: #1c1a19;
+}
```

Comment:
- [ ] Kindly use `flex` to align this section, since flexbox is always better for a 1-dimensional layout compared to a `grid`. It'll allow you to give them the required space more easily, and also enable you to wrap if needed. It'll also be much easier to layout when you switch to styling the desktop version.

Reply:
also done here